playbooks/bgp_l3_fabric/clos-fabric.yaml

```
@@ -1,43 +1,42 @@
 ---
-- hosts: datacenter
+- name: "Configure BGP clos fabric options after configuring prerequisite options"
+  hosts: datacenter
+  gather_facts: false
   connection: httpapi
-  gather_facts: no
-  collections:
-     - dellemc.enterprise_sonic
   tasks:
-  - name: Create Interfaces
-    sonic_interfaces:
-      config: "{{sonic_interfaces}}"
-    when: sonic_interfaces is defined and sonic_interfaces
+    - name: Create Interfaces
+      dellemc.enterprise_sonic.sonic.sonic_interfaces:
+        config: "{{ sonic_interfaces }}"
+      when: sonic_interfaces is defined and sonic_interfaces
 
-  - name: Create L3 Interfaces
-    sonic_l3_interfaces:
-      config: "{{sonic_l3_interfaces}}"
-    when: sonic_l3_interfaces is defined and sonic_l3_interfaces
+    - name: Create L3 Interfaces
+      dellemc.enterprise_sonic.sonic..sonic_l3_interfaces:
```

There are two periods before the final item in the path in here when there should be one:
```suggestion
      dellemc.enterprise_sonic.sonic.sonic_l3_interfaces:
```